### PR TITLE
PCHR-1925: Fixed Additional documnent or task break new assignment screen UI

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
## Problem
Additional document or task break new assignment screen UI

pchr-1925

### Fixed by #167  
Change markup for the structure to fix up the UI

resolved-pchr-1925

### Fixed By this PR:
Restructure Html markup so that the form does not go out of the modal. 
On selecting larger value for Task type in newly added task, the task type select option field remains with constant width.
constant-width-ui-select

### CODE REVIEW

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html [hunk 1]
```diff
@@ -128,18 +129,20 @@ <h4 class="modal-title">New Assignment</h4>
...
-                        <span class="input-group-btn" ng-if="activity.isAdded">
-                          <a class="btn btn-default btn-danger" ng-click="removeActivity(taskList, $index)">
-                            <span class="fa fa-remove" aria-hidden="true"></span>
-                          </a>
-                        </span>
                     </div>
                   </td>
+                  <td align="center">
+                    <span class="input-group-btn" ng-if="activity.isAdded">
+                      <a class="btn btn-default btn-danger" ng-click="removeActivity(taskList, $index)">
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html]: Are you sure that you need both `btn-default` and `btn-danger`?

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html [hunk 2]
```diff
@@ -128,18 +129,20 @@ <h4 class="modal-title">New Assignment</h4>
...
                         <span class="input-group-addon fa fa-calendar"></span>
-                        <span class="input-group-btn" ng-if="activity.isAdded">
-                          <a class="btn btn-default btn-danger" ng-click="removeActivity(taskList, $index)">
-                            <span class="fa fa-remove" aria-hidden="true"></span>
-                          </a>
-                        </span>
                     </div>
                   </td>
+                  <td align="center">
+                    <span class="input-group-btn" ng-if="activity.isAdded">
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html]: Why do you need this `input-group-btn` class for a simple button?
Also it seems that the span can be removed and the `a` can be directly placed inside `td`

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html [hunk 3]
```diff
@@ -128,18 +129,20 @@ <h4 class="modal-title">New Assignment</h4>
...
                           ng-click="!isDisabled && activity.create ? dpOpen($event, 'task'+$index) : return;" ng-disabled="isDisabled || !activity.create">
                         <span class="input-group-addon fa fa-calendar"></span>
-                        <span class="input-group-btn" ng-if="activity.isAdded">
-                          <a class="btn btn-default btn-danger" ng-click="removeActivity(taskList, $index)">
-                            <span class="fa fa-remove" aria-hidden="true"></span>
-                          </a>
-                        </span>
                     </div>
                   </td>
+                  <td align="center">
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html]: Dont use non-css properties to style. Instead you can use bootstrap class to do the same.

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html (hunk 1 above)
Comment (bmnepali) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html]: `btn-default` seems to add little shadow, which seems expected as this was the previously return code.

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html (hunk 3 above)
Comment (bmnepali) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html]: Right! will do.

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html (hunk 2 above)
Comment (bmnepali) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html]: Don't need `input-group-btn` as we are placing the button separately. Also will place `<a ...> ... </a>` directly in `td`.

**Pull request merged**